partcad/src/partcad/wrappers/wrapper_brep.py

```
@@ -0,0 +1,42 @@
+sys.path.append(os.path.dirname(__file__))
+import wrapper_common
+
+
+def process(path):
+    try:
+        shape = TopoDS_Shape()
+        builder = BRep_Builder()
+        brep_tools = BRepTools()
+
+        if not brep_tools.Read_s(shape, path, builder):
+            raise Exception("BREP file could not be loaded")
```

Ruff (0.8.2)

Abstract `raise` to an inner function

(TRY301)